redis/files/redis-2.6.conf.jinja

```diff
@@ -48,86 +43,86 @@ save 300 10
 save 60 10000
 {% endif -%}
 
-stop-writes-on-bgsave-error {{ redis.get('stop-writes-on-bgsave-error', 'yes') }}
+stop-writes-on-bgsave-error {{ redis_settings.stop_writes_on_bgsave_error }}
 
-rdbcompression {{ redis.get('rdbcompression', 'yes') }}
-rdbchecksum {{ redis.get('rdbchecksum', 'yes') }}
+rdbcompression {{ redis_settings.rdbcompression }}
+rdbchecksum {{ redis_settings.rdbchecksum }}
 
-dbfilename {{ redis.get('dbfilename', 'dump.rdb') }}
-dir {{ redis.get('root_dir', '/var/lib/redis') }}
+dbfilename {{ redis_settings.dbfilename }}
+dir {{ redis_settings.db_dir }}
```

Comment:
Please change this to ``redis_settings.root_dir`` and make appropriate changes elsewhere.